JSDemos/Demos/DataGrid/RecordPaging/Angular/app/app.component.ts

```diff
@@ -11,20 +11,28 @@ if(!/localhost/.test(document.location.host)) {
 @Component({
     selector: 'demo-app',
     templateUrl: 'app/app.component.html',
-    providers: [Service]
+    providers: [Service],
+    styleUrls: ['app/app.component.css']
 })
 export class AppComponent {
-    dataSource: Company[];
-
+    dataSource: Employee[];
+    readonly displayModes = ["full", "compact"];
```

Comment:
Maybe to use `const` above AppComponent class?

Reply:
It's used in binding